src/amo/components/AddonMoreInfo.js

```diff
@@ -16,21 +17,25 @@ export class AddonMoreInfoBase extends React.Component {
   render() {
     const { addon, i18n } = this.props;
 
-    let homepage = addon.homepage ? addon.homepage.trim() : null;
-    if (homepage && !homepage.match(/^https?:\/\//)) {
-      homepage = `http://${homepage}`;
-    }
+    const homepage = trimAndAddProtocolToUrl(addon.homepage);
+    const supportUrl = trimAndAddProtocolToUrl(addon.support_url);
 
     return (
       <Card className="AddonMoreInfo" header={i18n.gettext('More information')}>
         <dl className="AddonMoreInfo-contents">
-          {addon.homepage ? <dt>{i18n.gettext('Website')}</dt> : null}
-          {homepage ? (
-            <dd>
-              <a href={homepage}
-                ref={(ref) => { this.homepageLink = ref; }}>
-                {homepage.replace(/^https?:\/\//, '')}
-              </a>
+          {addon.homepage ? <dt>{i18n.gettext('Add-on Links')}</dt> : null}
```

Comment:
The dt will only be shown when addon.homepage is available. Presumably it should now be on the basis of either a homepage link or support url?